index.html

```diff
@@ -3002,13 +3031,15 @@
         </dt>
```

Comment:
Above this line, it says "ECMA-262 6th Edition, The ECMAScript 2015 Language Specification", but probably it should say "ECMAScript" or "The ECMAScript Language Specification" or similar.